### ISSUE TITLE
TBC - Dungeons - Boss reworks

### ISSUE THREAD

**Issue by killerwife** [cmangos/issues#3444]
List of bosses that are in stock AI state and need a rework to reach quality parity. CombatAI bosses are not a priority.

Black morass
- [x] Aeonus
- [x] Chrono Lord Deja
- [x] Temporus

Old Hillsbrad Foothills
- [x] Lieutenant Drake
- [x] Captain Skarloc
- [x] Epoch Hunter

Auchenai Crypts
- [x] Exarch Maladaar
- [x] Shirrak

Mana Tombs
- [x] Pandemonius
- [x] Yor
- [x] Tavarok

Sethekk Halls

- [x] Anzu
- [x] Darkweaver Syth

Shadow Labyrinth
- [x] Ambassador Hellmaw
- [x] Blackheart the Inciter
- [x] Grandmaster Vorpil
- [x] Murmur

Slave Pens
- [x] Mennu the Betrayer
- [x] Rokmar the Crackler
- [x] Quagmirran

Steamvaults
- [x] Hydromancer Thespia
- [x] Mekgineer Steamrigger
- [x] Warlord Kalithresh

Shattered Halls
- [x] Grand Warlock Nethekurse
- [x] Omrogg
- [x] Warchief Kargath Bladefist

Arcatraz
- [x] Dalliah
- [x] Zereketh the Unbound
- [x] Wrath-Scryer Soccothrates
- [x] Harbinger Skyriss

Botanica
- [x] High Botanist Freywinn
- [x] Laj
- [x] Thorngrin the Tender
- [x] Warp Splinter

Mechanar
- [x] Mechano Lord Capacitus
- [x] Nethermancer Sepethrea
- [x] Pathaleon the Calculator

World
- [x] Doom Lord Kazzak
- [x] Doomwalker

The Eye
- [x] Kael'Thas

Serpentshrine Cavern
- [x] Hydross the Unstable

Basically a progress bar for myself or anyone who wants to help.

**Comment by killerwife** [comment]
TODOs:
Lieutenant drake - figure out exact trigger for exploding shot

**Comment by miraco** [comment]
> TODOs: Lieutenant drake - figure out exact trigger for exploding shot

Seems to just be timer based, just engaged this boss and stood still for some time, got hit by explosing shot 2 already.

![1](https://github.com/cmangos/issues/assets/6261245/fef1d4ce-501d-476c-900f-0673364e83bf)
![2](https://github.com/cmangos/issues/assets/6261245/c8fc3c49-7a92-42a1-a20f-ee6c2aac0301)
![3](https://github.com/cmangos/issues/assets/6261245/99d13202-1280-4ae3-af87-95c400c9b0bd)
![4](https://github.com/cmangos/issues/assets/6261245/0d0bcc4c-48da-4a79-8a84-ec063bcd837d)

**Comment by killerwife** [comment]
Weird, I checked all my sniffs from legion and classic tbc, I had not a single spellcast in there. Was this normal or hc.

**Comment by miraco** [comment]
Heroic

**Comment by killerwife** [comment]
Guess check if it also occurs on normal then and let me know.

**Comment by AnonXS** [comment]
also seems he casts it on topaggro not 2nd to last?

and as it does stun he might also do distancing? spell has 0-50y range!

**Comment by miraco** [comment]
No distancing, but yes cast is on top aggro 
https://youtu.be/0sdzuDOuhso?t=60

**Comment by killerwife** [comment]
You did not actually dmg the boss in the vid, which on blizz end despite being combat doesnt mean you are eligible for other abilities. So I would not rule out random targeting just yet :D

**Comment by miraco** [comment]
I did heal my pet, this should put me into combat, and i also hitted him here https://youtu.be/0sdzuDOuhso?t=37

/edit anonxs ~4 casts ~50% random target but 4 times it goes to pet 1/2^4 ~6% for this to happen so it's very likely this ability is cast on topaggro only, timers are just not right init timer more like 30-60 and going from there.

**Comment by killerwife** [comment]
Note, Durnholde Veterans use a Knockdown spells which is unconfirmed anywhere else and spell id is post classic launch. So official has definitely been tampered with.

**Comment by miraco** [comment]
> I did heal my pet, this should put me into combat, and i also hitted him here https://youtu.be/0sdzuDOuhso?t=37
> 
> /edit anonxs ~4 casts ~50% random target but 4 times it goes to pet 1/2^4 ~6% for this to happen so it's very likely this ability is cast on topaggro only, timers are just not right init timer more like 30-60 and going from there.

Just to keep this info somewhere.
I tested it now for a bit longer (without recording) and he never used it random, only on main target. Timer seems to be rly high tho.



> Note, Durnholde Veterans use a Knockdown spells which is unconfirmed anywhere else and spell id is post classic launch. So official has definitely been tampered with.

If you talk about the "Kidney Shot" listed on wow wiki, they dont used in on my tests on wotlk classic ptr. Aslo tried to find old videos and none of them showed that they would use other spells then backstab/sinister strike.
so our current implementation seems fine, maybe just need to adjust timers someday.

**Comment by niefenger** [comment]
Blizzard is resisted by monsters and becomes an instant skill. It does not read a bar and consumes mana,

The Kesardi Cavalry Trainer trains the trolls of their own tribe to worship their reputation in order to learn
